app/src/main/java/com/islaharper/dawnofandroid/domain/model/ApiRequest.kt

```diff
@@ -0,0 +1,9 @@
+package com.islaharper.dawnofandroid.domain.model
+
+import kotlinx.serialization.Serializable
+
+// Used to verify token for Google One Tap signin
+@Serializable
+data class ApiRequest(
```

Comment:
This class is named `ApiRequest` , which makes it sound like it is the base class for all other ApiRequests, but it currently only seems to be used for making a token verification request. What do you think about finding a clearer name for this one?